packages/nx-plugin/src/generators/library/schema.json

```diff
@@ -12,7 +12,26 @@
         "index": 0
       },
       "x-prompt": "What name would you like to use for the library? (WITHOUT @fluentui-contrib/)"
+    },
+    "owner": {
+      "type": "string",
+      "description": "Team owning the library. Will be written in CODEOWNERS",
+      "$default": {
+        "$source": "argv",
+        "index": 1
+      },
+      "x-priority": "important",
+      "x-prompt": {
+        "message": "Which team will own this library",
+        "type": "list",
+        "items": [
+          "@microsoft/cxe-red",
```

Comment:
because this is contrib/extensions repo owners might vary. would be great if we could add "another" option so user can specify a custom team/owner if necessary. 

as a start I'd remove the x-prompt as type of list to force user always manually add owning team.

Reply:
That makes sense!